### PR TITLE
Workaround hardcoding komodo and pexporter image name

### PR DESCRIPTION
**Pull request by lgarciaaco**
This is the less intrusive solution I can think of, hardcode komodo and postgres exporter IS.

Fixes https://github.com/syndesisio/fuse-online-install/issues/170

### CODE REVIEW

**Comment by avano** [comment]
I'm on it

File: update_ocp.sh [hunk 1]
```diff
@@ -330,6 +330,13 @@ update_imagestreams() {
   for image in $images; do
       local is=${IMAGE_NAME_PREFIX}-$image
       eval tag_image=\$tag_${image}
+      if [ "$image" == "postgres_exporter" ]; then
+	is="fuse-postgres-exporter"
```
Comment (avano) [inline update_ocp.sh]: the correct image stream name is `postgres_exporter` and the correct tag expected by the operator is `v0.4.7`, this is what the operator expects:

```
        from:
          kind: ImageStreamTag
          name: postgres_exporter:v0.4.7
```

File: update_ocp.sh [hunk 2]
```diff
@@ -330,6 +330,13 @@ update_imagestreams() {
...
+      fi
+
+      if [ "$image" == "komodo" ]; then
+	is="data-virtualization-server-rhel7"
```
Comment (avano) [inline update_ocp.sh]: the correct name for komodo is `fuse-komodo-server` and the correct tag is (sadly) `latest`:

```
        from:
          kind: ImageStreamTag
          name: fuse-komodo-server:latest

```

Comment (heiko-braun) [reply update_ocp.sh]: Yes, we had to accept this for 7.4

**Comment by lgarciaaco** [comment]
Closing this PR and sending another one, after talking to @heiko-braun the solution would be not to upgrade komodo and postgres exporter

**Pull request closed without merging**